docs/architecture/hotspot-makers/become-a-maker/hotspot-ble-services.mdx

```diff
@@ -15,7 +15,7 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
 
 <LegacyContentBanner />
 
-The Helium Hotspot provides a Bluetooth Low Energy\(BLE\) interface for users to configure several
+The Helium Hotspot provides a Bluetooth Low Energy \(BLE\) interface for users to configure several
```

Comment:
Since you've added the space, the parens don't need to be escaped anymore.
```suggestion
The Helium Hotspot provides a Bluetooth Low Energy (BLE) interface for users to configure several
```